### DISCUSSION TITLE
Discussions vs Issues?

### DISCUSSION THREAD

**Discussion by AlgorithmsAreCool** [dotnet/runtime#37628] [General]
I noticed we can have discussions here now. Is there guidance on what should qualify a discussion vs an issue for this repo?

**Comment by stephentoub** [comment]
cc: @danmosemsft

**Reply by danmoseley** [reply]
I think we're all figuring them out. Maybe...an issue if it's more likely than not that it would be resolved by a code change in the repo, otherwise a discussion? What do you suggest?

**Comment by danmoseley** [comment]
It would be interesting to collect observations about Discussions vs Issues here. Eg., when it's appropriate to transfer them. 

One thing of note is that we don't have a labeling or notification system. So e.g if you open a question about Crypto, you have to hope someone capable of answering it can find it -- or look up the owners in https://github.com/dotnet/runtime/blob/master/docs/area-owners.md

**Reply by AlgorithmsAreCool** [reply]
The lack of labeling seems problematic, especially this is a large repo with many sub-concerns.
The discussion feature also seems a little confused. The "Mark as answer" feature makes it seem like a Q/A section. 
But it also has threaded conversations which seems geared towards longer well...discussions.

I'd love this for the dotnet/csharplang repo however...

**Reply by samsp-msft** [reply]
Personally I see discussions as more somewhere to ask questions, get advice, that are not as likely to result in code changes, unlike issues which are for bugs, feature requests etc.
You can move items from issues to discussion and refer to either by number - which is nice.

> One thing of note is that we don't have a labeling or notification system. So e.g if you open a question about Crypto, you have to hope someone capable of answering it can find it -- or look up the owners in https://github.com/dotnet/runtime/blob/master/docs/area-owners.md

We should give that feedback to github. Should this be a free-form tagging system, or pick from a menu like issues?

**Reply by danmoseley** [reply]
@samsp-msft I suggest they simply enable adding existing labels. Whether they do or not will obviously depend on whether it's widespread feedback..
